app/helpers/admin_helper.rb

```diff
@@ -22,4 +22,19 @@ def action_button(name, link, method: :post, class_style: 'btn btn-link')
                 form: {data: { turbo: true, turbo_confirm: t('admin.turbo_confirm', name: name), turbo_frame: '_top'}}
 
   end
+
+  def portal_config_container
+    attribute_to_delete = ["context", "links", "@id", "@type"]
+    attribute_to_not_modify = ["id", "federated_portals", "fundedBy", "keyword", "numberOfArtefacts", "metrics", "numberOfClasses", "numberOfIndividuals", "numberOfProperties", "numberOfAxioms", "numberOfObjectProperties", "numberOfDataProperties", "numberOfLabels", "numberOfDeprecated", "numberOfUsingProjects", "numberOfEndorsements", "numberOfMappings", "numberOfUsers", "numberOfAgents" ]
+    root_url = "#{rest_url}/?display=all"
+    attributes_catalog = LinkedData::Client::HTTP.get(root_url, {display: :all})
```

Comment:
you need to create a model in the UI, see the existent to get inspiration https://github.com/ontoportal-lirmm/ontologies_api_ruby_client/tree/master/lib/ontologies_api_client/models
With you can use
`your_model.get`
`your_model.update`